Misc/NEWS.d/next/Library/2020-11-05-13-32-41.bpo-42264.r4KYUU.rst

```diff
@@ -0,0 +1,2 @@
+``sqlite3.OptimizedUnicode`` has been deprecated. It has been undocumented
+and obsolete since Python 3.3.
```

Comment:
I suggest to copy your good What's New entry:

```

  ``sqlite3.OptimizedUnicode`` has been undocumented and obsolete since Python
  3.3, when it was made an alias to :class:`str`.  It is now deprecated,
  scheduled for removal in Python 3.12.
```

Reply:
Agreed! Fixed in 2507fd03301a47ee4df6eb45cbe99afbf322ab12